test_utils/edx_platform_mocks/django_filters/rest_framework.py

```diff
@@ -0,0 +1,17 @@
+""""Mocked filterbackend class"""
+
+
+class DjangoFilterBackend:   # pylint: disable=too-few-public-methods
+    """
+    A mock filter backend that mimics the behavior of DjangoFilterBackend.
+    """
+    def filter_queryset(
+        self, request, queryset, view
+    ):  # pylint: disable=unused-argument, no-self-use
+        """
+        This is the mock implementation of filter_queryset.
+        In a real scenario, it would filter the queryset based on the request parameters.
+
+        In this mock class, it doesn't filter the queryset at all and just returns it.
+        """
+        return queryset  # Simply return the queryset as-is (no filtering).
```

Comment:
@tehreem-sadat , please remove this mock completely. I've added `django-filter` to [tests requirements](https://github.com/nelc/futurex-openedx-extensions/pull/132). Just rebase on `main` and your tests will pass without this mock